scripts/registerParachain/extrinsics.js

```diff
@@ -0,0 +1,147 @@
+function findSudoError(events) {
+    let sudoError;
+    events.forEach((record) => {
+        const { event } = record;
+        const types = event.typeDef;
+        if (event.method === 'Sudid') {
+            return event.data.forEach((data, index) => {
+                if (types[index].type == 'DispatchResult' && data.isError) {
+                    sudoError = data.toJSON().err.module;
```

Comment:
Just because I stumbled upon it today: `isError` apparently is deprecated, use `isErr` (& `asErr`) instead.
```suggestion
                if (types[index].type == 'DispatchResult' && data.isErr) {
                    sudoError = data.asErr.asModule.toJSON();
```

Reply:
here is the reference re/ deprecation: https://github.com/polkadot-js/api/blob/4498f596073163fc7bccf205fa61ee575833c77a/packages/types/src/interfaces/system/types.ts#L162-L171

Reply:
Thanks a lot for the hint. Fixed in 39f50fa!